src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable no-undef */
+export const Person = ({ name, age, partner }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+    {age ? <p className="Person__age">{`I am ${age}`}</p> : Null}
+    {isMarried ? (
+      <p className="Person__partner">
+        {`${partner} is my {
+      person.sex === 'm' ? 'wife' : 'nusband'}`}
```

Comment:
looks broken, try to fix it and provide the demo link in the PR description

Reply:
https://verjane23.github.io/react_person/